packages/sirius-web/frontend/sirius-web-application/src/views/project-browser/list-projects-area/ListProjectsArea.tsx

```diff
@@ -41,7 +41,8 @@ export const ListProjectsArea = ({}: ListProjectsAreaProps) => {
     endCursor: null,
   });
 
-  const { data, loading, refreshProjects } = useProjects(state.startCursor, state.endCursor, state.pageSize);
+  // TODO: Add extension point somewhere to contribute new filters
```

Comment:
Don't contribute TODOs, use Github issues if you need to remind yourself to do something

Reply:
Removed